dotcom-rendering/src/web/layouts/StandardLayout.tsx

```diff
@@ -580,6 +580,9 @@ export const StandardLayout = (props: WebProps | AppProps) => {
 									hasStarRating={
 										typeof article.starRating === 'number'
 									}
+									supportsDarkMode={
+										renderingTarget === 'Apps'
+									}
```

Comment:
Rather than deciding this at the layout level, then passing `supportsDarkMode` to every component, maybe we should only pass renderingTarget around (as this will probably propegate across a lot of the components anyway), then have the `darkModeCss` function take renderingTarger instead supportsDarkMode and then it becomes the source-of-truth on which rendering targets support dark mode or not (one place)